pkg/teleporter/relayer.go

```diff
@@ -362,6 +362,8 @@ func CreateBaseRelayerConfig(
 		SourceBlockchains:      []*config.SourceBlockchain{},
 		DestinationBlockchains: []*config.DestinationBlockchain{},
 		MetricsPort:            metricsPort,
+		DBWriteIntervalSeconds: 10,
```

Comment:
do you think we need to hardcode this? 
there is a separate PR to use only released version?

Reply:
added constants. yes, finally I found that increasing the relayer dep on go.mod, updates this struct,
which cause the conf file to be badly generated with incorrect default values